cluster/src/main/scala/com/linkedin/norbert/jmx/JMX.scala

```diff
@@ -51,7 +51,7 @@ object JMX extends Logging {
 
   def unregister(mbean: ObjectInstance) = try {
     mbeanServer.unregisterMBean(mbean.getObjectName)
-    //synchronized { map.remove(mbean.getObjectName.getCanonicalName) } We treat the map as a mapping
+    synchronized { map.remove(mbean.getObjectName.getCanonicalName) } //We treat the map as a mapping
```

Comment:
cloud-ncs dails if i dont uncomment this. I pulled the latest norbert, and cloud ncs doesn't pass its tests.